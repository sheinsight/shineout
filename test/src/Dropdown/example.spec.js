import exampleTest from '../../example'

exampleTest('Dropdown', {
  ignore: 'example-7-base.tsx',
})
