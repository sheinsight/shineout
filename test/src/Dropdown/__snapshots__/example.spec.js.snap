// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-1-base.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <a>
          Link to Google
        </a>
        <a>
          Disabled
        </a>
      </div>
      <a>
        <span>
          Submenu
        </span>
        <span>
          <svg>
            <path />
          </svg>
        </span>
      </a>
    </div>
    <a>
      Home
    </a>
    <a>
      Message
    </a>
  </div>
  <button>
    <span>
      Dropdown
    </span>
    <span>
      <svg>
        <path />
      </svg>
    </span>
  </button>
</div>
`;

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-2-hover.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <a>
          optic 1
        </a>
      </div>
      <a>
        <span>
          First
        </span>
        <span>
          <svg>
            <path />
          </svg>
        </span>
      </a>
    </div>
    <div>
      <div>
        <div>
          <div>
            <a>
              topic 3
            </a>
          </div>
          <a>
            <span>
              topic 2
            </span>
            <span>
              <svg>
                <path />
              </svg>
            </span>
          </a>
        </div>
      </div>
      <a>
        <span>
          Second
        </span>
        <span>
          <svg>
            <path />
          </svg>
        </span>
      </a>
    </div>
  </div>
  <button>
    <span>
      Hover
    </span>
    <span>
      <svg>
        <path />
      </svg>
    </span>
  </button>
</div>
`;

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-3-position.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            First
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                link4
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            Second
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Right Top
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            First
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                link4
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            Second
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Bottom Left
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            First
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
              <span>
                link4
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            Second
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Bottom Right
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            First
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
              <span>
                link4
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            Second
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        <svg>
          <path />
        </svg>
      </span>
      <span>
        Left Top
      </span>
    </button>
  </div>
  <br />
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            First
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                link4
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            Second
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Right Bottom
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            First
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                link4
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            Second
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Top Left
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            First
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
              <span>
                link4
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            Second
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Top Right
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            First
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
              <span>
                link4
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            <svg>
              <path />
            </svg>
          </span>
          <span>
            Second
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        <svg>
          <path />
        </svg>
      </span>
      <span>
        Left Bottom
      </span>
    </button>
  </div>
  <br />
  <div>
    <div>
      <div>
        <div>
          <a>
            link1
          </a>
          <a>
            link2
          </a>
        </div>
        <a>
          <span>
            First
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <div>
        <div>
          <a>
            link3
          </a>
          <div>
            <div>
              <a>
                link5
              </a>
              <a>
                link6
              </a>
            </div>
            <a>
              <span>
                link4
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </a>
          </div>
        </div>
        <a>
          <span>
            Second
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
    </div>
    <button>
      <span>
        Auto Position
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
</div>
`;

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-4-items.tsx] 1`] = `
<div>
  <div>
    <a>
      1
    </a>
    <a>
      2
    </a>
    <a>
      3
    </a>
    <a>
      4
    </a>
    <a>
      5
    </a>
    <a>
      6
    </a>
    <a>
      7
    </a>
    <a>
      8
    </a>
    <a>
      9
    </a>
    <a>
      10
    </a>
    <a>
      11
    </a>
    <a>
      12
    </a>
    <a>
      13
    </a>
    <a>
      14
    </a>
    <a>
      15
    </a>
    <a>
      16
    </a>
    <a>
      17
    </a>
    <a>
      18
    </a>
    <a>
      19
    </a>
    <a>
      20
    </a>
    <a>
      21
    </a>
    <a>
      22
    </a>
    <a>
      23
    </a>
    <a>
      24
    </a>
    <a>
      25
    </a>
    <a>
      26
    </a>
    <a>
      27
    </a>
    <a>
      28
    </a>
    <a>
      29
    </a>
    <a>
      30
    </a>
  </div>
  <button>
    <span>
      Dropdown
    </span>
    <span>
      <svg>
        <path />
      </svg>
    </span>
  </button>
</div>
`;

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-5-split.tsx] 1`] = `
<div>
  <button>
    <span>
      Left
    </span>
  </button>
  <button>
    <span>
      Center
    </span>
  </button>
  <div>
    <div>
      <a>
        First
      </a>
      <a>
        Second
      </a>
    </div>
    <button>
      <span />
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
</div>
`;

exports[`Dropdown[snapshot] Snapshot test: Dropdown[site/pages/components/Dropdown/example-6-type.tsx] 1`] = `
<div>
  <div>
    <span>
      type: 
    </span>
    <div>
      <div>
        <div>
          <span>
            primary
          </span>
          <a>
            <svg>
              <path />
            </svg>
          </a>
        </div>
        <div>
          <span>
            Data not found
          </span>
        </div>
      </div>
    </div>
    <span>
      size: 
    </span>
    <div>
      <div>
        <div>
          <span>
            default
          </span>
          <a>
            <svg>
              <path />
            </svg>
          </a>
        </div>
        <div>
          <span>
            Data not found
          </span>
        </div>
      </div>
    </div>
    <label>
      <input />
      <i />
      <span>
        outline
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        disabled
      </span>
    </label>
  </div>
  <div>
    <div>
      <div>
        <div>
          <a>
            Link
          </a>
          <a>
            Disabled
          </a>
        </div>
        <a>
          <span>
            Submenu
          </span>
          <span>
            <svg>
              <path />
            </svg>
          </span>
        </a>
      </div>
      <a>
        Message
      </a>
    </div>
    <button>
      <span>
        Dropdown
      </span>
      <span>
        <svg>
          <path />
        </svg>
      </span>
    </button>
  </div>
</div>
`;
